training/trillium/Mixtral-8x7B-Pytorch/GCE/host.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+DOCKER_IMAGE=us-central1-docker.pkg.dev/deeplearning-images/reproducibility/pytorch-tpu-mixtral:v0
+
+worker_id=$(curl -s "http://metadata.google.internal/computeMetadata/v1/instance/attributes/agent-worker-number" -H 'Metadata-Flavor: Google')
+
+cat >> /dev/null <<EOF
+EOF
+
+stdbuf -oL bash <<-PIPE_EOF 2>&1 | sed "s/^/[worker $slice_id:$worker_id] /g" | tee runlog
```

Comment:
`slice_id` is not extracted. I can't find how to get the value. Maybe we can skip it.

Reply:
Done